tldw_Server_API/app/api/v1/endpoints/connectors.py

```
@@ -73,24 +73,24 @@ async def oauth_callback(
+    try:
+        memberships = current_user.get("org_memberships") or []
+        # pick active org or first membership
+        org_id = memberships[0]["org_id"] if memberships else 1
+        pol = await get_policy(db, org_id)
+        if not pol:
+            pol = get_default_policy_from_env(org_id)
+        role = str(current_user.get("role", "member")).lower()
+        required = str(pol.get("account_linking_role", "admin")).lower()
+        # Admin bypass
+        if role != "admin" and required and role != required:
+            raise HTTPException(status_code=403, detail="Account linking not permitted for your role")
```

Ruff (0.14.6)

Abstract `raise` to an inner function

(TRY301)